macos/Podfile.lock

```diff
@@ -163,7 +169,8 @@ SPEC CHECKSUMS:
   sqlite3: 73b7fc691fdc43277614250e04d183740cb15078
   sqlite3_flutter_libs: 06a05802529659a272beac4ee1350bfec294f386
   url_launcher_macos: d2691c7dd33ed713bf3544850a623080ec693d95
+  video_player_avfoundation: 02011213dab73ae3687df27ce441fbbcc82b5579
 
 PODFILE CHECKSUM: 353c8bcc5d5b0994e508d035b5431cfe18c1dea7
 
-COCOAPODS: 1.13.0
+COCOAPODS: 1.15.0
```

Comment:
This CocoaPods bump is fine, but should be its own commit because it's logically independent of the other things happening. See `git log --stat -p -G COCOAPODS` for past examples.

I suspect that on your machine if you just do `flutter run` for iOS and macOS targets, starting from main, it'll already make these two changes.